bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1085,6 +1085,11 @@ public function objectbrickUpdateAction(Request $request)
                 $brickDef->setLayoutDefinitions($layout);
             }
 
+            $event = new GenericEvent($this, [
+                'brickDefinition' => $brickDef
+            ]);
+            \Pimcore::getEventDispatcher()->dispatch(AdminEvents::CLASS_OBJECTBRICK_UPDATE_DEFINITION, $event);
+
```

Comment:
fetch `brickDefinition` argument from event before save for the changes to be utilized. i.e. https://github.com/pimcore/pimcore/blob/8f09d1164447af032dc3fa972b8cdb58c9d2146b/bundles/AdminBundle/Controller/Admin/Document/DocumentController.php#L189

Reply:
@dvesh3 Hey Divesh, you are right. PR-Updated. Should be fixed now.